docker/Dockerfile.cpu

```diff
@@ -2,8 +2,10 @@ FROM ubuntu:latest
 
 COPY install.sh /
 
-RUN /install.sh && pip uninstall --yes mxnet && pip install mxnet-mkl
+RUN /install.sh && pip uninstall --yes mxnet && pip install mxnet-mkl && mkdir /mxnet_model_server
```

Comment:
i double checked. currently this *does* install a usable version of mkl. yay!

Reply:
Cool...